src/components/Login/LoginPanel.tsx

```diff
@@ -31,7 +33,7 @@ export interface LoginPanelInterface {
 }
 
 const LoginPanel = (props: LoginPanelInterface) => {
```

Comment:
I'd recommend `{linkAccount, displayGoogle, displayFacebook, displayTwitter}: LoginPanelInterface` for object destructuring. Helps to avoid using `props.` in code to reduce chances of typos and avoid clutter.

Reply:
As far as I could tell, I still had to define type for each of these, resulting in a really long line. I think leaving it as is would be best (unless I'm doing it wrong)

Reply:
<img width="958" alt="image" src="https://user-images.githubusercontent.com/39249792/179387016-d729cefc-b3c3-427e-a1ab-9c41d0c0d87d.png">

^this worked for me. `props.` does have the advantage you can easily ready which variables are props in your code so up to you if you wanna change it.

Reply:
Ah I see, yeah I think this is better. Implemented 👍🏻